lab01-activity.Rmd

```diff
@@ -277,3 +303,4 @@ At the end of every lab, replace TODO with DONE to the left of each task to indi
 |TODO| Miniconda is installed|
```

Comment:
Jazmin, for next assignments after you have done each of these parts change TODO to DONE.